refinerycms-blog.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.test_files        = `git ls-files -- spec/*`.split("\n")
 
   # Runtime dependencies
+  s.add_dependency    'rails',                 '~> 4.0'
```

Comment:
We don't need(and want) to add dependency to rails in this extension as`refinerycms-core` already define it : https://github.com/refinery/refinerycms/blob/master/core/refinerycms-core.gemspec#L5